HyperVExtension/src/HyperVExtension/CommunicationWithGuest/GuestKvpChannel.cs

```diff
@@ -88,13 +110,60 @@ private void SendMessage(string name, string value, CancellationToken stoppingTo
         }
     }
 
-    public List<IResponseMessage> WaitForResponseMessages(string responseId, TimeSpan timeout, bool expectProgressResponse, CancellationToken stoppingToken)
+    /// <summary>
+    /// Delete Hyper-V KVP message from the VM.
+    /// Best effort. Log error if failed, but don't throw.
+    /// </summary>
+    /// <param name="name">Hyper-V KVP name.</param>
+    private void RemoveKvpItem(string name)
+    {
+        try
+        {
+            using ManagementClass kvpExchangeDataItem = new ManagementClass(_scope, new ManagementPath("Msvm_KvpExchangeDataItem"), null);
+            using ManagementObject dataItem = kvpExchangeDataItem.CreateInstance();
+            dataItem["Data"] = string.Empty;

```

Comment:
Should "Data" and all the other string literals be declared as a constant somewhere? Though I'm not seeing who is consuming these values that are being set.  Is the consumer in a different git repo?

Reply:
You are right, it's not consumed in the same way. It's in this repo, but it's consumed as reading a registry value. Hyper-V service creates those registry values on guest OS. Those literals are only used in this file, so I'll leave it as it is for now.